src/dev/flang/be/jvm/Intrinsix.java

```diff
@@ -607,16 +752,7 @@ else if (in.equals("fuzion.sys.internal_array.setel"))
         });
 
     put(new String[]
-      { "fuzion.java.array_get",
-        "fuzion.java.array_length",
-        "fuzion.java.array_to_java_object0",
-        "fuzion.java.call_c0",
-        "fuzion.java.call_s0",
-        "fuzion.java.call_v0",
-        "fuzion.java.get_field0",
-        "fuzion.java.get_static_field0",
-        "fuzion.java.java_string_to_string",
-        "fuzion.java.string_to_java_object0"
+      {
```

Comment:
please add a comment here that the names of not-implemented intrinsics should go here.